app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
```

Comment:
Although now I think about it, perhaps a "middle ground" solution could be something along the lines of:

- If a user navigates to `/accordion`, show all sections collapsed
- If a user navigates to `/accordion#section1`, jump to section1 and expand just that section
- If a user navigates to `/accordion#`, restore session from session storage
- Toggling links in the accordion will simply set the address bar to `/accordion#`. If a user wants to share a link, they'll have to copy the link from the accordion header (which will have `href=/accordion#section1` - this will give good middle-click behaviour, and allow said sharing)

cc @carvil